src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/SealightsCLIBuildStep.java

```diff
@@ -250,7 +250,7 @@ public CLIRunner createCLIRunner(CommandMode commandMode){
                     configView.getBuildName(),configView.getAdditionalArguments(),null);
         }else {
             Properties properties = PropertiesUtils.toProperties(commandMode.getAdditionalArguments());
-            String labId= (properties.get("labid")!= null)? properties.get("labid").toString():null;
+            String labId= (commandMode.getLabId()!= null)? commandMode.getLabId() : properties.get("labid").toString();
```

Comment:
Are you sure that's correct? 
What happens if someone puts a space?
Did you test that end to end?

Reply:
Space is treated like any other string (same goes for all input parameters) but it printed with quotes so it will be visible to the user.
I tested it and it sent to the backend with an empty labid (response 200)

Reply:
Maybe we should trim all the values?